api/swagger.yaml

```diff
@@ -4443,6 +4443,8 @@ paths:
       responses:
         200:
           description: "no error"
+        404:
+          description: "repository not found"
```

Comment:
@bfirsh Nice catch,  thx for your attention 👍  !  And I've updated the schema definition to swagger.yaml.